Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -808,8 +808,8 @@ YUI.add('ez-platformuiapp', function (Y) {
                     view: "serverSideView",
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
                 }, {
-                    name: "adminRole",
-                    regex: /\/admin\/(pjax%2Frole.*)/,
+                    name: "adminLanguage",
```

Comment:
Yes, I think we all missed that new route was created with already existing name: https://github.com/ezsystems/PlatformUIBundle/pull/344/files#diff-4e897328f624be930d091d1ce688a3ffR784
So I think that Andre's change is OK.